libs/remix-ui/file-explorer/src/lib/file-explorer.tsx

```diff
@@ -280,29 +304,32 @@ export const FileExplorer = (props: FileExplorerProps) => {
     }
   }
 
-  const deletePath = async (path: string) => {
+  const deletePath = async (path: string | string[]) => {
```

Comment:
I think it will be better to create a new function that accepts `string[]` and uses the implementation of `deletePath`.